src/menu/menu.c

```diff
@@ -17,13 +17,13 @@
 #include "common/mem.h"
 #include "common/nodename.h"
 #include "common/scaled_font_buffer.h"
+#include "common/scene-helpers.h"
 #include "common/string-helpers.h"
 #include "labwc.h"
 #include "menu/menu.h"
 #include "node.h"
 #include "theme.h"
 
-#define MENUWIDTH (110)
 #define MENU_ITEM_PADDING_Y (4)
 #define MENU_ITEM_PADDING_X (7)
```

Comment:
item padding should also be moved to theme

Reply:
Agree

Reply:
Moved. New theme vars:
- `menu.items.padding.x`
- `menu.items.padding.y`